federation/src/graphql/api/1_1/resolver/PublicKeyResolver.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { Resolver } from 'type-graphql'
+// eslint-disable-next-line camelcase
+import { PublicKeyResolver as PublicKeyResolver_1_0 } from '../../1_0/resolver/PublicKeyResolver'
```

Comment:
i thought relative paths are not allowed in imports?

Reply:
Let's create a Ticket for this!

Reply:
good idea

Reply:
- [ ] 🚀 [Feature] Federation: introduce linting rules

Reply:
#3064